pkg/application/inject/fuse/mount_point_script.go

```diff
@@ -82,6 +82,9 @@ func (s *Injector) injectCheckMountReadyScript(pod common.FluidObject, runtimeIn
 }
 
 func (s *Injector) ensureScriptConfigMapExists(namespace string) (*poststart.ScriptGeneratorForApp, error) {
+	if len(namespace) == 0 {
```

Comment:
Why can't namespace  be found?

Reply:
https://github.com/kubernetes/client-go/blob/master/rest/request.go#L872